src/core.ts

```diff
@@ -38,13 +39,6 @@ addEventListener("beforeunload", async () => {
   await kv.close();
 });
 
-// OAuth 2.0 session
-export interface OAuthSession {
```

Comment:
I move this into types, so that the internal oauth2_client.ts can reference it without depending on this core module from which it doesn't required anything else.